cmd/promtool/main.go

```diff
@@ -171,16 +176,96 @@ func checkRules(t cli.Term, filename string) (int, error) {
 		return 0, fmt.Errorf("is a directory")
 	}
 
+	rgs, errs := rulefmt.ParseFile(filename)
+	if errs != nil {
+		return 0, tsdb.MultiError(errs)
```

Comment:
Same about import `tsdb` for this and probably better returning the actual slice so the errors can be logged as one per line. `tsdb.MultiError` will just concat them with `;` which will be impossible to debug with.